op-wheel/commands.go

```diff
@@ -59,12 +60,16 @@ var (
 		Value:    "http://localhost:8551/",
 		EnvVars:  prefixEnvVars("ENGINE"),
 	}
+	EngineJWT = &cli.StringFlag{
+		Name:    "engine.jwt-secret",
+		Usage:   "JWT secret used to authenticate Engine API communication with.",
+		EnvVars: prefixEnvVars("ENGINE_JWT_SECRET"),
+	}
 	EngineJWTPath = &cli.StringFlag{
```

Comment:
Hmm, doesn't the new flag var need to be passed to the app init somewhere?

Reply:
you're right, fixed!